modules/media/ajax/FileUpload.php

```diff
@@ -347,13 +349,15 @@ function toSelect($options, $item, $item2)
 {
     $selectOptions = [];
 
-    $optionsValue = $item;
+    $optionsVal = $item;
```

Comment:
This block could be written more concisely like this:

```php
$optionsVal = $item2 ?? $item;
```

Reply:
@um4r12 why is this being changed ?

Reply:
@ridz1208 Before I had added the DDE filter in the query, the left join would produce some null records, and this resulted into empty select options for one of the fields. So I added an if statement to ensure that the select options were not empty, but with the current variable names, I was exceeding the 80 char limit with the indentation. Shortened it to meet that requirement.

Reply:
@nicolasbrossard done